gateway-service/src/test/java/org/zowe/apiml/gateway/controllers/GatewayHomepageControllerTest.java

```diff
@@ -120,6 +172,15 @@ public void givenApiCatalogWithEmptyInstances_whenHomePageCalled_thenHomePageMod
 
     @Test
     public void givenApiCatalogInstance_whenHomePageCalled_thenHomePageModelShouldContain() {
+        ServiceInstance authserviceInstance = new DefaultServiceInstance("instanceId", "serviceId",
+            "host", 10000, true);
+        when(discoveryClient.getInstances("zosmf")).thenReturn(
```

Comment:
Following four lines are duplicates. What about moving them in the private method?